From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@trinadhk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

dominicbetts committed:
Keep central and suite in sync

dominicbetts committed:
More synchronization

ChrisGMsft committed:
Update iot-hub-devguide-quotas-throttling.md

ChrisGMsft committed:
Update max size description

ChrisGMsft committed:
Update direct method payload size

ChrisGMsft committed:
Update with higher direct method limits

klaaslanghout committed:
Update with customer issue "moving AD tenant" in troubleshooting

alkohli committed:
dotnet jobs and default path files

alkohli committed:
fixed image link

alkohli committed:
overview and automation files

alkohli committed:
ui file

dariagrigoriu committed:
Serviceability update.

Updated **Symptom**: RPC failed; result=22, HTTP code = 5xx.

Commit:
updating acrolinx score

dariagrigoriu committed:
Merge pull request #2 from SyntaxC4-MSFT/dariag-patch1

Updating acrolinx scores

Commit:
Merge pull request #4 from MicrosoftDocs/master

merge from Microsoft

Commit:
Update prebuilt entity supported languages table

neilpeterson committed:
initial doc

neilpeterson committed:
updates

neilpeterson committed:
initial doc

neilpeterson committed:
toc

bwren committed:
Update api versions

alkohli committed:
pulled all the changes from Vivek

alkohli committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into SSDMS2

alkohli committed:
new file

bwren committed:
specify resource type

alkohli committed:
more changes

alkohli committed:
added images

alkohli committed:
minor changes

Commit:
add image

Commit:
add more screenshots

rayne-wiselman committed:
updating vmware new features

Commit:
Update how-to-vscode-develop-csharp-module.md

rayne-wiselman committed:
fixing couple of links

Commit:
Update how-to-vscode-develop-csharp-module.md

alkohli committed:
minor changes

alkohli committed:
added image

alkohli committed:
Acrolinx changes

neilpeterson committed:
update

v-shils committed:
Update how-to-vscode-develop-csharp-module.md

neilpeterson committed:
updates

alkohli committed:
resized images

Commit:
fix numbered list

Commit:
fix minor issues

Commit:
minor

neilpeterson committed:
updates

neilpeterson committed:
updates

rayne-wiselman committed:
Update site-recovery-vmware-to-azure-manage-configuration-server.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

rayne-wiselman committed:
Update support-matrix-vmware-to-azure.md

rayne-wiselman committed:
Update site-recovery-vmware-to-azure-manage-configuration-server.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

rayne-wiselman committed:
Update site-recovery-vmware-to-azure-manage-configuration-server.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

neilpeterson committed:
updates

alkohli committed:
removed references to media ssets

alkohli committed:
removed media asset references

dariagrigoriu committed:
Update app-service-deploy-local-git.md

Updated numbering.

alkohli committed:
minor fix

Commit:
task 853: Update steps

Commit:
task 853: Fix link

Commit:
add more images

rayne-wiselman committed:
Update concepts-vmware-to-azure-architecture.md

rayne-wiselman committed:
Update tutorial-vmware-to-azure.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

rayne-wiselman committed:
Update site-recovery-vmware-to-azure-manage-configuration-server.md

rayne-wiselman committed:
Update tutorial-vmware-to-azure.md

rayne-wiselman committed:
Update how-to-deploy-configuration-server.md

curtand committed:
first ten of 40 SEO updates

curtand committed:
last four articles for SEO test

neilpeterson committed:
typo

linda33wj committed:
Update SP property description

linda33wj committed:
Clarify empty folder won't be copied

linda33wj committed:
Update connector category

linda33wj committed:
Update connector category

linda33wj committed:
Update connector category

rayne-wiselman committed:
updating prepare azure tutorial

rayne-wiselman committed:
Merge branch 'rayne-january10' of https://github.com/rayne-wiselman/azure-docs-pr into rayne-january10

Commit:
Removed preview for service tags.

Commit:
Fix

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into seo0112-1

curtand committed:
H1 fix

mimig1 committed:
Cosmos DB: Adding ODBC server list

mimig1 committed:
Typo fix

bprakash committed:
Update apache-domain-joined-configure-using-azure-adds.md

mimig1 committed:
changed format

mimig1 committed:
acrolink fixes

spelluru committed:
Updated titles

bprakash committed:
Update apache-domain-joined-manage.md

bprakash committed:
Update apache-domain-joined-manage.md

Removed SSH option

neilpeterson committed:
updated role to reader

alkohli committed:
modified images per PR reviewer

Commit:
new document

alkohli committed:
Changes from PR review, TOC modified

Commit:
fixing formatting

alkohli committed:
Changes from PR review

Commit:
formatting. For some reason there's an extra line between two items and not others. Seeing if this fixes it.

bwren committed:
dates

bwren committed:
Move legacy log search api

Commit:
fixing typos

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

bwren committed:
change header

mimig1 committed:
updates

anrothMSFT committed:
updated author and available formats

Android export was added as a feature in Dec 2017. Adding information about Android export and a link to our Android sample app.

Commit:
fixing merge conflicts

Commit:
fixing merge conflict

tfitzmac committed:
freshness update

mimig1 committed:
removed prereqs

Commit:
Merge pull request #4 from SyntaxC4-MSFT/dariag-patch1

fixing merge conflicts

alkohli committed:
minor changes in 1st para

Commit:
Update app-service-deploy-local-git.md

neilpeterson committed:
updates

Commit:
Merge pull request #31122 from DeniseMak/1-9-prebuilt-lang

Update prebuilt entity supported languages table

Commit:
Merge pull request #31261 from bwren/oms-views

Management solutions update api versions

ktoliver committed:
Merge pull request #30878 from dariagrigoriu/patch-1

Serviceability update.

PRMerger17 committed:
Merge pull request #31523 from neilpeterson/aks-sp-update

updated role to reader

Commit:
Merge pull request #30498 from klaaslanghout/patch-8

Update with customer issue "moving AD tenant" in troubleshooting

ktoliver committed:
Merge pull request #31533 from bwren/la-searchapi

Move legacy log search api

barbkess committed:
Consolidate downlevel-clients into auditing overview

barbkess committed:
redirect deleted file

Commit:
Merge pull request #31543 from tfitzmac/0116refresh

freshness update

Commit:
task 853: Update screen shots per latest GUI

markgalioto committed:
adding backup article for setting up backup while creating VM

rmca14 committed:
Merge pull request #31238 from neilpeterson/azure-disks-pv

Azure Disks - Kubernetes Persistent Volume

barbkess committed:
acrolinx fixes

Commit:
Merge pull request #31516 from spelluru/adftitles0116

Updated titles

SnehaGunda committed:
Adding video

PRMerger10 committed:
Merge pull request #31548 from linda33wj/master

Tweak ADF copy content

Commit:
Merge pull request #31534 from dkkapur/master

fixing typos

PRMerger17 committed:
Merge pull request #31538 from anrothMSFT/patch-1

updated author and available formats

Commit:
Merge pull request #31522 from bprakash/patch-16

Update apache-domain-joined-manage.md

PRMerger10 committed:
Merge pull request #31520 from bprakash/patch-15

Update apache-domain-joined-manage.md

PRMerger14 committed:
Merge pull request #31514 from bprakash/patch-14

Update apache-domain-joined-configure-using-azure-adds.md

PRMerger12 committed:
Merge pull request #31503 from dominicbetts/suite-central-sync

Suite central sync

PRMerger14 committed:
Merge pull request #31501 from jimdial/patch-61

Removed preview for service tags.

SnehaGunda committed:
Adding TOC item

v-alje committed:
Merge pull request #31536 from MicrosoftDocs/FromPublicRepo

From public repo

SnehaGunda committed:
Adding TOC item

vladvino committed:
Added Timestamp and Elapsed

PRMerger15 committed:
Merge pull request #31551 from SnehaGunda/CL

Adding video

spelluru committed:
Updated monitoring screenshots

Commit:
Merge pull request #31552 from vladvino/patch-2

Added Timestamp and Elapsed

spelluru committed:
updated the monitoring image for Tutorial 3

spelluru committed:
Updated monitor images for Tutorial 2

PRMerger10 committed:
Merge pull request #29793 from ChrisGMsft/patch-14

Update with higher direct method limits

PRMerger20 committed:
Merge pull request #29792 from ChrisGMsft/patch-13

Update direct method payload size

PRMerger19 committed:
Merge pull request #29791 from ChrisGMsft/patch-12

Added new Direct Method limits

GitHubber17 committed:
Merge pull request #31454 from dksimpson/dean-853

task 853: Update steps

GitHubber17 committed:
Update backup-during-vm-creation.md

GitHubber17 committed:
Merge pull request #31550 from markgalioto/ab-1-16-2018

adding backup article for setting up backup while creating VM

v-jasowells committed:
Update BingVoiceOutput.md

MightyPen committed:
'East US 2' tertiary IP added, as * NOTE.

2.167.104.0
Per PM Dhruv Malik.

v-jasowells committed:
Update BingVoiceOutput.md

PRMerger12 committed:
Merge pull request #31560 from v-jaswel/master

Update BingVoiceOutput.md

bprakash committed:
Update apache-domain-joined-configure-using-azure-adds.md

Commit:
Merge pull request #31562 from bprakash/patch-17

Update apache-domain-joined-configure-using-azure-adds.md

MightyPen committed:
Escape the asterisk in front of the NOTE, below the table.

PRMerger20 committed:
Merge pull request #31561 from MightyPen/patch-2

'East US 2' tertiary IP added, as * NOTE.

dominicbetts committed:
Fix typo

PRMerger17 committed:
Merge pull request #31563 from dominicbetts/fix-threat-typo

Fix typo

dominicbetts committed:
Embed video

Commit:
Merge pull request #31566 from dominicbetts/add-customize-video

Embed video

dominicbetts committed:
Update deployment article

dominicbetts committed:
Update CF details in What are article

dominicbetts committed:
Update CF details in What are article

dominicbetts committed:
Update CF FAQ

Commit:
per marketing, should always be 'Azure Cosmos DB'

spelluru committed:
added a link to the video

PRMerger13 committed:
Merge pull request #31569 from spelluru/adfvideo0117

added a link to the video

dominicbetts committed:
Fix link

spelluru committed:
added the Resource Manager Template section

PRMerger12 committed:
Merge pull request #31515 from mimig1/01162018odbc

Cosmos DB: Adding ODBC server list

itechedit committed:
Merge pull request #30632 from alkohli/SSDMS3

StorSimple Data Manager Release PR#3 - Go Live Jan 17

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into adfssisarm0117

spelluru committed:
updated

ktoliver committed:
Merge pull request #31285 from shizn/master

Add screenshots to iot edge dev tutorials

PRMerger17 committed:
Merge pull request #31570 from spelluru/adfssisarm0117

added the Resource Manager Template section

rmca14 committed:
Merge pull request #31568 from dominicbetts/cf-updates

Connected factory updates

itechedit committed:
Merge pull request #30626 from alkohli/SSDMS1

StorSimple Data Manager Release PR#1 - Go Live Jan 17

itechedit committed:
Merge pull request #30631 from alkohli/SSDMS2

StorSimple Data Manager Release PR#2 - Go Live Jan 17

itechedit committed:
Merge pull request #31292 from rayne-wiselman/rayne-january10

updating vmware new features

itechedit committed:
Merge pull request #31528 from Blackmist/freshness

new document

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Merge pull request #31549 from barbkess/audit

Consolidate downlevel-clients into auditing overview

v-alje committed:
Merge pull request #31580 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #31486 from curtand/seo0112-1

first ten of 40 SEO updates

PRMerger14 committed:
Merge pull request #31497 from curtand/seo0112-4

last six articles for SEO test

itechedit committed:
Merge pull request #31555 from spelluru/getstarted0116

Monitor activities screen has changed
